pennylane/transforms/qcut/cutcircuit.py

```diff
@@ -32,15 +31,33 @@
 from .utils import find_and_place_cuts, fragment_graph, replace_wire_cut_nodes
 
 
-@batch_transform
+def _cut_circuit_expand(
+    tape: QuantumTape,
+    use_opt_einsum: bool = False,
+    device_wires: Optional[Wires] = None,
+    max_depth: int = 1,
+    auto_cutter: Union[bool, Callable] = False,
+    **kwargs,
+) -> (Sequence[QuantumTape], Callable):
+    """Main entry point for expanding operations until reaching a depth that
+    includes :class:`~.WireCut` operations."""
+    # pylint: disable=unused-argument
+
+    def processing_fn(res):
+        return res[0]
+
+    return [_qcut_expand_fn(tape, max_depth, auto_cutter)], processing_fn
+
+
+@partial(transform, expand_transform=_cut_circuit_expand)
```

Comment:
have we considered adding the setter syntax? I'm not the biggest fan, but users might be more familiar with it. I see that we have it for `custom_qnode_transform` already (works as a decorator, like line 407 below)

Reply:
Yes I think we will keep it as it is because we agreed on that at the start of the project.